b2sdk/sync/sync.py

```diff
@@ -96,6 +96,32 @@ class KeepOrDeleteMode(Enum):
 
 
 class Synchronizer(object):
+    """
+    Copies multiple files from source to destination.  Optionally
```

Comment:
```suggestion
    Copies multiple "files" from source to destination.  Optionally
```
those are objects, really, but we call them files here.